examples/dts/sub-orchestrations-with-fan-out-fan-in/orchestrator.py

```diff
@@ -0,0 +1,44 @@
+import os
+from azure.identity import DefaultAzureCredential
+from durabletask import client
+from durabletask.azuremanaged.client import DurableTaskSchedulerClient
+
+# Read the environment variable
+taskhub_name = os.getenv("TASKHUB")
+
+# Check if the variable exists
+if taskhub_name:
+    print(f"The value of TASKHUB is: {taskhub_name}")
+else:
+    print("TASKHUB is not set. Please set the TASKHUB environment variable to the name of the taskhub you wish to use")
```

Comment:
Again, I suggest we remove this requirement. We can assume default values for emulator usage if these environment variables aren't sent.